### DISCUSSION TITLE
IMX577 Driver support

### DISCUSSION THREAD

**Discussion by tirankas** [OE4T/meta-tegra#743] [General]
Hello guys,
Wondering if we have the IMX577 driver support for the Jetson Nano/Xavier.
I didn't find anything...

Thanks

**Comment by ichergui** [comment]
Hey @tirankas 
`Linux-tegra` version 4.9 doesn't include driver `IMX577`, maybe you have to ask the provider of the Camera.
In general when you buy a camera you can have the Linux driver with.

**Reply by tirankas** [reply]
Thanks @ichergui 
Well, this is usually the case but not here, camera vendor doesn't provide any driver in this case.I would probably end up taking the driver from the vanilla kernel and modify DT per the Jetson.

**Reply by madisongh** [reply]
@tirankas You might want to consider working with one of the [NVIDIA camera partners](https://developer.nvidia.com/embedded/jetson-partner-supported-cameras) for this.

**Comment by tirankas** [comment]
I know the partner approach and we were working with one previously using the ar1335 sensor...
However, given their price tag for a module and given the sensor price (160$ for a module whole sensor costs 10$ max), this is not an option...
Frankly, I don't understand where the price delta comes from, the module is so so simple, I don't understand where this comes from.
We are building on hundreds of thousands of sensors in our roadmap and the price difference is huge!
